java/ql/src/change-notes/2023-10-06-thread-resource-abuse-arithexpr.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I don't think we need change notes for experimental queries, feel free to remove this file.

Reply:
Thanks! Removed the change notes. 

Should be good to go. 😀